hoodie-client/src/main/java/com/uber/hoodie/HoodieReadClient.java

```diff
@@ -128,7 +131,7 @@ private void assertSqlContext() {
 
     assertSqlContext();
     JavaPairRDD<HoodieKey, Optional<String>> keyToFileRDD = index
-        .fetchRecordLocation(hoodieKeys, hoodieTable);
+        .fetchRecordLocation(hoodieKeys, jsc, hoodieTable.getMetaClient());
```

Comment:
I want to remove the SparkContext from Index since we are already passing it to the table and apart from a method in BloomIndex, none of the index actually requires that. I already synced with @bvaradar on this after explaining to him that we are moving index under the table which he was unaware of when he made this comment.

Reply:
Okay understood.